phpunit.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="./vendor/phpunit/phpunit/phpunit.xsd"
+         bootstrap="./vendor/autoload.php"
+         colors="true"
+         executionOrder="random"
+         failOnWarning="true"
+         failOnRisky="true"
+         verbose="true"
```

Comment:
```suggestion
         executionOrder="random"
         beStrictAboutChangesToGlobalState="true"
         beStrictAboutOutputDuringTests="true"
         beStrictAboutCoversAnnotation="true"
         forceCoversAnnotation="true"
         failOnRisky="true"
         failOnEmptyTestSuite="true"
         verbose="true"
```

a few more config parameters I think we should use, you can see what they do here: https://phpunit.readthedocs.io/en/9.5/configuration.html#the-phpunit-element